cmd/destroy/destroy.go

```diff
@@ -281,7 +304,7 @@ func (dc *destroyCommand) getDestroyer(ctx context.Context, opts *Options) (dest
 				return nil, err
 			}
 		}
-		runInRemote := !isRemote && (destroyImage != "" || opts.RunInRemote)
+		runInRemote := shouldRunInRemote(opts) && (destroyImage != "" || opts.RunInRemote)
 
 		if runInRemote {
```

Comment:
```suggestion
		if shouldRunInRemote(opts) {
```
`(destroyImage != "" || opts.RunInRemote)` is checked inside `shouldRunInRemote(opts)` function